src/convertToObject.js

```diff
@@ -12,7 +12,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObj = {};
+
+  sourceString
+    .split(';')
+    .forEach((style, ind, arr) => {
+      const property = style
+        .split(':')
+        .map(elem => elem.trim());
+
+      stylesObj[property[0]] = property[1];
+    });
+
+  delete stylesObj[''];
```

Comment:
check property value before assigning it yo object, so you won't need to delete it later